### ISSUE TITLE
Razor syntax not working, MacOS

### ISSUE THREAD

**Issue by RickDoll** [microsoft/vscode-dotnettools#1042]
Type: <b>Bug</b>

## Issue Description ##
On MacOS, using C# dev Kit, razor ryntax is broken. The project builds fine.

Uninstalling C# dev kit resolves the issue, reinstalling C# dev Kit, issue returns.

## Steps to Reproduce ##
Open a blazor project in VSCode on MacOS Arm.

### C# log ###
<details>--------------------------------------------------------------------------------
Razor.VSCode version 7.0.0-preview.24165.1
--------------------------------------------------------------------------------
Razor's log level is currently set to 'Trace'
 - To change Razor's log level set 'razor.server.trace' and then restart VSCode.
 - To report issues invoke the 'Report a Razor issue' command via the command palette.
-----------------------------------------------------------------------------------------------------------------------------

Using dotnet configured on PATH
[Client - 9:28:08 AM] Razor language server path: /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razor/rzls
[Client - 9:28:08 AM] Server arguments --logLevel 0 --projectConfigurationFileName project.razor.vscode.bin --DelegateToCSharpOnDiagnosticPublish true --UpdateBuffersForClosedDocuments true --telemetryLevel all --sessionId b2d2f483-46a7-40c7-9c54-1724d4ff36041712064487797 --telemetryExtensionPath /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razortelemetry/Microsoft.VisualStudio.DevKit.Razor.dll
[Client - 9:28:08 AM] Starting Razor Language Server...
You must install or update .NET to run this application.

App: /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razor/rzls
Architecture: arm64
Framework: 'Microsoft.NETCore.App', version '8.0.1' (arm64)
.NET location: /usr/local/share/dotnet

The following frameworks were found:
  6.0.20 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  7.0.9 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  8.0.0 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.1&arch=arm64&rid=osx-arm64&os=osx.14
[Error - 9:28:08 AM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Info  - 9:28:08 AM] Connection to server got closed. Server will restart.
true
[Error - 9:28:08 AM] Razor Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
You must install or update .NET to run this application.

App: /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razor/rzls
Architecture: arm64
Framework: 'Microsoft.NETCore.App', version '8.0.1' (arm64)
.NET location: /usr/local/share/dotnet

The following frameworks were found:
  6.0.20 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  7.0.9 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  8.0.0 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.1&arch=arm64&rid=osx-arm64&os=osx.14
[Error - 9:28:08 AM] Server initialization failed.
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Error - 9:28:08 AM] Razor Language Server client: couldn't create connection to server.
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Error - 9:28:08 AM] Restarting server failed
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Client - 9:28:08 AM] (Error) Failed when activating Razor VSCode.
Cannot call write after a stream was destroyed
Stack Trace:
Error: Cannot call write after a stream was destroyed
    at /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/dist/extension.js:2:2097336
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[Info  - 9:28:08 AM] Connection to server got closed. Server will restart.
true
You must install or update .NET to run this application.

App: /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razor/rzls
Architecture: arm64
Framework: 'Microsoft.NETCore.App', version '8.0.1' (arm64)
.NET location: /usr/local/share/dotnet

The following frameworks were found:
  6.0.20 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  7.0.9 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  8.0.0 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.1&arch=arm64&rid=osx-arm64&os=osx.14
[Error - 9:28:08 AM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Info  - 9:28:08 AM] Connection to server got closed. Server will restart.
true
[Error - 9:28:08 AM] Razor Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
You must install or update .NET to run this application.

App: /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razor/rzls
Architecture: arm64
Framework: 'Microsoft.NETCore.App', version '8.0.1' (arm64)
.NET location: /usr/local/share/dotnet

The following frameworks were found:
  6.0.20 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  7.0.9 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  8.0.0 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.1&arch=arm64&rid=osx-arm64&os=osx.14
[Error - 9:28:08 AM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Info  - 9:28:08 AM] Connection to server got closed. Server will restart.
true
[Error - 9:28:08 AM] Razor Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 9:28:08 AM] Restarting server failed
  Message: Pending response rejected since connection got disposed
  Code: -32097 
You must install or update .NET to run this application.

App: /Users/rick/.vscode/extensions/ms-dotnettools.csharp-2.22.5-darwin-arm64/.razor/rzls
Architecture: arm64
Framework: 'Microsoft.NETCore.App', version '8.0.1' (arm64)
.NET location: /usr/local/share/dotnet

The following frameworks were found:
  6.0.20 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  7.0.9 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  8.0.0 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.1&arch=arm64&rid=osx-arm64&os=osx.14
[Error - 9:28:08 AM] Server initialization failed.
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Error - 9:28:08 AM] Razor Language Server client: couldn't create connection to server.
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Error - 9:28:08 AM] Restarting server failed
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Error - 9:28:08 AM] Restarting server failed
  Message: Cannot call write after a stream was destroyed
  Code: -32099 
[Error - 9:28:08 AM] The Razor Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/MainLayout.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/MainLayout.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/MainLayout.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/MainLayout.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/NavMenu.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/NavMenu.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/NavMenu.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/NavMenu.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/RedirectToLogin.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/RedirectToLogin.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/RedirectToLogin.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/RedirectToLogin.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/App.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/App.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/App.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/App.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Authentication.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Authentication.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Authentication.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Authentication.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Home.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Home.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Home.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Pages/Home.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/LoginDisplay.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/LoginDisplay.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/LoginDisplay.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/Layout/LoginDisplay.razor__virtual.cs
[Client - 9:28:10 AM] File '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/_Imports.razor' didn't exist in the Razor document list. This is likely because it's from outside the workspace.
[Client - 9:28:10 AM] Notifying document '/Users/rick/Development/Source/GlobalWizard/GlobalWizard/_Imports.razor' changed 'added'
[Client - 9:28:10 AM] (Warning) razor.provideDynamicFileInfo failed with CodeExpectedError: cannot open virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/_Imports.razor__virtual.cs. Detail: Unable to resolve resource virtualCSharp-razor:/Users/rick/Development/Source/GlobalWizard/GlobalWizard/_Imports.razor__virtual.cs
</details>

## Environment information ##

**VSCode version**: 1.87.2
**C# Extension**: 2.22.5
**Using OmniSharp**: true

<details><summary>Dotnet Information</summary>
.NET SDK:
 Version:           8.0.100
 Commit:            57efcf1350
 Workload version:  8.0.100-manifests.d75fd3e4

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  14.3
 OS Platform: Darwin
 RID:         osx-arm64
 Base Path:   /usr/local/share/dotnet/sdk/8.0.100/

.NET workloads installed:
 Workload version: 8.0.100-manifests.d75fd3e4
 [maui]
   Installation Source: SDK 8.0.100
   Manifest Version:    8.0.3/8.0.100
   Manifest Path:       /usr/local/share/dotnet/sdk-manifests/8.0.100/microsoft.net.sdk.maui/8.0.3/WorkloadManifest.json
   Install Type:        FileBased

Host:
  Version:      8.0.0
  Architecture: arm64
  Commit:       5535e31a71

.NET SDKs installed:
  6.0.412 [/usr/local/share/dotnet/sdk]
  7.0.306 [/usr/local/share/dotnet/sdk]
  8.0.100 [/usr/local/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.20 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.9 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.20 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.9 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

Other architectures found:
  None

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|Folder Name|
|---|---|---|---|
|al|ms-dynamics-smb|13.0.971907|ms-dynamics-smb.al-13.0.971907|
|al-code-outline|andrzejzwierzchowski|13.0.61|andrzejzwierzchowski.al-code-outline-13.0.61|
|azure-account|ms-vscode|0.11.6|ms-vscode.azure-account-0.11.6|
|azure-dev|ms-azuretools|0.8.1|ms-azuretools.azure-dev-0.8.1|
|azure-repos|ms-vscode|0.38.0|ms-vscode.azure-repos-0.38.0|
|blazortemplatewizard|TelerikInc|2.14.0|telerikinc.blazortemplatewizard-2.14.0|
|blazorwasm-companion|ms-dotnettools|1.1.4|ms-dotnettools.blazorwasm-companion-1.1.4|
|copilot|GitHub|1.177.0|github.copilot-1.177.0|
|copilot-chat|GitHub|0.13.1|github.copilot-chat-0.13.1|
|crs-al-language-extension|waldo|1.5.31|waldo.crs-al-language-extension-1.5.31|
|csdevkit|ms-dotnettools|1.4.29|ms-dotnettools.csdevkit-1.4.29-darwin-arm64|
|csharp|ms-dotnettools|2.22.5|ms-dotnettools.csharp-2.22.5-darwin-arm64|
|csharpextensions|kreativ-software|1.7.3|kreativ-software.csharpextensions-1.7.3|
|data-workspace-vscode|ms-mssql|0.5.0|ms-mssql.data-workspace-vscode-0.5.0|
|debugpy|ms-python|2024.2.0|ms-python.debugpy-2024.2.0-darwin-arm64|
|dotnet-interactive-vscode|ms-dotnettools|1.0.5164010|ms-dotnettools.dotnet-interactive-vscode-1.0.5164010|
|intellicode-api-usage-examples|VisualStudioExptTeam|0.2.8|visualstudioexptteam.intellicode-api-usage-examples-0.2.8|
|java|redhat|1.28.1|redhat.java-1.28.1-darwin-arm64|
|java-decompiler|dgileadi|0.0.4|dgileadi.java-decompiler-0.0.4|
|jupyter|ms-toolsai|2024.2.0|ms-toolsai.jupyter-2024.2.0-darwin-arm64|
|jupyter-keymap|ms-toolsai|1.1.2|ms-toolsai.jupyter-keymap-1.1.2|
|jupyter-renderers|ms-toolsai|1.0.17|ms-toolsai.jupyter-renderers-1.0.17|
|lana|FinancialForce|1.14.1|financialforce.lana-1.14.1|
|mono|StepanVanzuriak|0.0.1|stepanvanzuriak.mono-0.0.1|
|mono-debug|ms-vscode|0.16.3|ms-vscode.mono-debug-0.16.3|
|mssql|ms-mssql|1.22.1|ms-mssql.mssql-1.22.1|
|mysql-shell-for-vs-code|Oracle|1.14.2|oracle.mysql-shell-for-vs-code-1.14.2-darwin-arm64|
|postman-for-vscode|Postman|0.19.1|postman.postman-for-vscode-0.19.1|
|prettier-vscode|esbenp|10.4.0|esbenp.prettier-vscode-10.4.0|
|python|ms-python|2024.2.1|ms-python.python-2024.2.1|
|quicktype|quicktype|12.0.46|quicktype.quicktype-12.0.46|
|remote-repositories|ms-vscode|0.40.0|ms-vscode.remote-repositories-0.40.0|
|remotehub|GitHub|0.62.0|github.remotehub-0.62.0|
|salesforce-soql-editor|allanoricil|1.8.0|allanoricil.salesforce-soql-editor-1.8.0|
|salesforce-vscode-slds|salesforce|1.4.8|salesforce.salesforce-vscode-slds-1.4.8|
|salesforcedx-vscode|salesforce|60.5.1|salesforce.salesforcedx-vscode-60.5.1|
|salesforcedx-vscode-apex|salesforce|60.5.1|salesforce.salesforcedx-vscode-apex-60.5.1|
|salesforcedx-vscode-apex-debugger|salesforce|60.5.1|salesforce.salesforcedx-vscode-apex-debugger-60.5.1|
|salesforcedx-vscode-apex-replay-debugger|salesforce|60.5.1|salesforce.salesforcedx-vscode-apex-replay-debugger-60.5.1|
|salesforcedx-vscode-core|salesforce|60.5.1|salesforce.salesforcedx-vscode-core-60.5.1|
|salesforcedx-vscode-lightning|salesforce|60.5.1|salesforce.salesforcedx-vscode-lightning-60.5.1|
|salesforcedx-vscode-lwc|salesforce|60.5.1|salesforce.salesforcedx-vscode-lwc-60.5.1|
|salesforcedx-vscode-soql|salesforce|60.5.1|salesforce.salesforcedx-vscode-soql-60.5.1|
|salesforcedx-vscode-visualforce|salesforce|60.5.1|salesforce.salesforcedx-vscode-visualforce-60.5.1|
|sftp|Natizyskunk|1.16.3|natizyskunk.sftp-1.16.3|
|sql-bindings-vscode|ms-mssql|0.4.0|ms-mssql.sql-bindings-vscode-0.4.0|
|sql-database-projects-vscode|ms-mssql|1.4.1|ms-mssql.sql-database-projects-vscode-1.4.1|
|vscode-azureappservice|ms-azuretools|0.25.2|ms-azuretools.vscode-azureappservice-0.25.2|
|vscode-azurecontainerapps|ms-azuretools|0.6.2|ms-azuretools.vscode-azurecontainerapps-0.6.2|
|vscode-azurefunctions|ms-azuretools|1.14.0|ms-azuretools.vscode-azurefunctions-1.14.0|
|vscode-azureresourcegroups|ms-azuretools|0.8.4|ms-azuretools.vscode-azureresourcegroups-0.8.4|
|vscode-azurestaticwebapps|ms-azuretools|0.12.2|ms-azuretools.vscode-azurestaticwebapps-0.12.2|
|vscode-azurestorage|ms-azuretools|0.15.3|ms-azuretools.vscode-azurestorage-0.15.3|
|vscode-azurevirtualmachines|ms-azuretools|0.6.5|ms-azuretools.vscode-azurevirtualmachines-0.6.5|
|vscode-cosmosdb|ms-azuretools|0.21.0|ms-azuretools.vscode-cosmosdb-0.21.0|
|vscode-dotnet-runtime|ms-dotnettools|2.0.3|ms-dotnettools.vscode-dotnet-runtime-2.0.3|
|vscode-eslint|dbaeumer|2.4.4|dbaeumer.vscode-eslint-2.4.4|
|vscode-gradle|vscjava|3.13.5|vscjava.vscode-gradle-3.13.5|
|vscode-html-format|mohd-akram|0.1.6|mohd-akram.vscode-html-format-0.1.6|
|vscode-java-debug|vscjava|0.57.0|vscjava.vscode-java-debug-0.57.0|
|vscode-java-dependency|vscjava|0.23.6|vscjava.vscode-java-dependency-0.23.6|
|vscode-java-pack|vscjava|0.25.15|vscjava.vscode-java-pack-0.25.15|
|vscode-java-test|vscjava|0.40.1|vscjava.vscode-java-test-0.40.1|
|vscode-jupyter-cell-tags|ms-toolsai|0.1.8|ms-toolsai.vscode-jupyter-cell-tags-0.1.8|
|vscode-jupyter-slideshow|ms-toolsai|0.1.5|ms-toolsai.vscode-jupyter-slideshow-0.1.5|
|vscode-maven|vscjava|0.44.0|vscjava.vscode-maven-0.44.0|
|vscode-node-azure-pack|ms-vscode|1.2.0|ms-vscode.vscode-node-azure-pack-1.2.0|
|vscode-nuget-gallery|patcx|0.0.24|patcx.vscode-nuget-gallery-0.0.24|
|vscode-nuget-package-manager|jmrog|1.1.6|jmrog.vscode-nuget-package-manager-1.1.6|
|vscode-pylance|ms-python|2024.3.2|ms-python.vscode-pylance-2024.3.2|
|vscode-xml|redhat|0.26.1|redhat.vscode-xml-0.26.1-darwin-arm64|
|vscodeintellicode|VisualStudioExptTeam|1.3.1|visualstudioexptteam.vscodeintellicode-1.3.1|
|vsliveshare|ms-vsliveshare|1.0.5918|ms-vsliveshare.vsliveshare-1.0.5918|;

</details>

Extension version: 1.4.29
VS Code version: Code 1.87.2 (863d2581ecda6849923a2118d93a088b0745d9d6, 2024-03-08T15:20:57.520Z)
OS version: Darwin arm64 23.3.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M2 Max (12 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|64.00GB (5.66GB free)|
|Process Argv|--crash-reporter-id 3ed61bf3-fd2d-492d-a5e7-e7afa69b1034|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:31000450
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
d34g3935:30971562
fegfb526:30981948
bg6jg535:30979843
ccp2r3:30993541
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
gd77d436:30999572

```

</details>

**Comment by RickDoll** [comment]
Not sure if you made a change, but the razor syntax in my blazor projects seem to be working now.

This showed up in my terminal on loading the project, so it seems as something was updated.

Installing Razor Telemetry Version = 7.0.0-preview.24168.4...
Platform: darwin, arm64

Downloading package 'Razor Language Server Telemetry (macOS ARM64)' (9343 KB).................... Done!
Validating download...
Integrity Check succeeded.
Installing package 'Razor Language Server Telemetry (macOS ARM64)'

**Comment by RickDoll** [comment]
Syntax is working now, but it seems like formatting shortcuts are not working for it, not sure if related.

**Comment by ryzngard** [comment]
Thanks for filing @RickDoll . Can you also add the razor log? It will be in the output window titled "Razor Log"

**Comment by ryzngard** [comment]
We made numerous fixes in `2.39.29` and this should be resolved.
